third_party/blink/renderer/core/brave_page_graph/page_graph.cc

```diff
@@ -1094,12 +1094,35 @@ NodeHTML* PageGraph::GetHTMLNode(const DOMNodeId node_id) const {
   return nullptr;
 }
 
-NodeHTMLElement* PageGraph::GetHTMLElementNode(const DOMNodeId node_id) const {
+NodeHTMLElement* PageGraph::GetHTMLElementNode(
+    absl::variant<blink::DOMNodeId, blink::Node*> node_var) {
+  blink::DOMNodeId node_id;
+  blink::Node* node = nullptr;
+  if (absl::holds_alternative<blink::DOMNodeId>(node_var)) {
```

Comment:
Why not just get the `node` here a la `blink::DOMNodeIds::NodeForId` given that we are doing that anyway down below?

Reply:
Gotcha. Could we add some comments here then explaining when that code path will be followed?

Reply:
> Gotcha. Could we add some comments here then explaining when that code path will be followed?

Sure, added some explanation into code. Thanks!